marray/tests/test_marray.py

```diff
@@ -164,26 +159,29 @@ def test_comparison_binary(f, dtype, seed=None):
     assert_equal(res, ref, seed)
 
 
-# @pytest.mark.parametrize("dtype", dtypes_integral + dtypes_real)
-# @pytest.mark.parametrize("f", inplace_arithmetic + inplace_bitwise)
-# def test_inplace(f, dtype, seed=None):
-#     marrays, masked_arrays, seed = get_arrays(2, dtype=dtype, seed=seed)
-#     e1 = None
-#     e2 = None
-#
-#     try:
-#         f(masked_arrays[0], masked_arrays[1])
-#     except Exception as e:
-#         e1 = e
-#     try:
-#         f(marrays[0], marrays[1])
-#     except Exception as e:
-#         e2 = e
-#
-#     if e1 or e2:
-#         assert str(e1) == str(e2)
-#     else:
-#         assert_equal(marrays[0], masked_arrays[0], seed)
+@pytest.mark.parametrize("dtype", dtypes_integral + dtypes_real)
+@pytest.mark.parametrize("f", inplace_arithmetic + inplace_bitwise)
+def test_inplace(f, dtype, seed=None):
```

Comment:
This test is pretty ugly. I have the `try`-`except`s mostly because the arrays are currently allowed to be of any *mutually* broadcastable shapes, so we get an error when the second array is not broadcastable to the shape of the first. I'm OK with that for now. Let's draft quickly and polish later.

Reply:
I'll have a look tomorrow to see if I have an idea how to improve, otherwise I agree to postpone polishing until later (the issue with that is that most of then time "later" either means "much later" or "never", so oftentimes I'd open an issue to remind me – though probably the project is small enough that this is not yet an issue?)

Reply:
Yeah, i will open an issue about it. But it's not just this function - every test function needs review and improvement. So I'm not very concerned about forgetting. This first draft of the tests is just a sanity check that the function basically works and is usable; a second round will be more thorough.